### ISSUE TITLE
make.autograph issue with violin plot

### ISSUE THREAD

**Issue by slin829** [ImmuneDynamics/Spectre#6]
Hi Tom. I think there's an issue with the violin plot with make.autograph. You get the following warning when violin = T, and you also don't get the underlying violin plot. 

"In max(data$density) : no non-missing arguments to max; returning -Inf Computation failed in `stat_ydensity()`:replacement has 1 row, data has 0"

**Comment by tomashhurst** [comment]
@slin829 Hmm interesting, was that from one of the demo workflows?

**Comment by slin829** [comment]
Yup it's part of simple discover work flow. Also tested with the example data set in the documentation. Same error. Might be related to ggplot's geom_violin? Not 100% sure

**Comment by tomashhurst** [comment]
Hmm OK we'll check it out!

**Comment by tomashhurst** [comment]
Hi @slin829 it works fine on my laptop. What do the plots look like? Are they _just_ missing the violin part?

**Comment by tomashhurst** [comment]
Hi @slin829 , did you get this working again in the end? If you like we can connect on zoom and we'll see if there is some straight forward fix.

**Comment by slin829** [comment]
Hi. Sorry I haven't had another go since then. My computer is getting replaced tomorrow, so I was thinking to wait until then, install and update all packages. Hopefully that will fix the issue! If not, I'll e-mail you! Thanks!!

**Comment by tomashhurst** [comment]
@slin829 did you get a chance to try it out again?

**Comment by slin829** [comment]
Yup I had another go this morning. Tested with the dataset in the documentation and it's still the same error. It's probably something to do with the sample size per group where the violin plot cannot be generated. Seems to be ok when I added more data points to the dataset.  

BUT a separate error came out that suggests some points may have been cut off from the plot.
"Warning message:
Removed 167 rows containing missing values (geom_violin). "
